examples/units_and_coordinates/north_offset_frame.py

```diff
@@ -0,0 +1,45 @@
+"""
+===============================================
+Creating an Offset frame using NorthOffsetFrame
+===============================================
+
+This is an example to show some possible usage of ``NorthOffsetFrame``.
+
+A common use for this is to create a frame derived from Heliographic, which
+has the north pole at some point of interest. In this new frame, lines of
+longitude form great circles radially away from the point, and lines of
+latitude measure angular distance from the point.
+In this example the new frame is shifted so the new north pole is at (20,
+20) in the Heliographic Stonyhurst frame. The new grid is overplotted in
+blue.
+
+"""
+
+##############################################################################
+# Import the required modules.
+import matplotlib.pyplot as plt
+import astropy.units as u
+
+from astropy.coordinates import SkyCoord
+from sunpy.coordinates import NorthOffsetFrame
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+##############################################################################
+# Creating the offset frame  
```

Comment:
```suggestion
# Now to create the offset frame.
```
There might need to be more text here but hopefully someone else can pitch in.

Reply:
Not extremely familiar with the library yet, I shall get to it when I get a better hang of it